UEBPrep/app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -1,40 +1,35 @@
 class OmniauthCallbacksController < Devise::OmniauthCallbacksController
-  def google_oauth2
-    @user = User.from_omniauth(request.env["omniauth.auth"])
 
+  def provider_login
```

Comment:
yeah this had been bugging me for a while, so I decided to take care of it with this go.